### ISSUE TITLE
Exporting auto-keras model and keras model doesn't act same

### ISSUE THREAD

**Issue by serengil** [keras-team/autokeras#612]
Hello, 

I fit a model in auto-keras and it gets 66.23% accuracy. Then I export this model in both auto-keras and regular keras format.

```
model.export_autokeras_model('best_autokeras_model.h5')
model.export_keras_model('best_keras_model.h5')
```

After then, I load these models in a different notebook. 

1- Loading auto-keras model gives the same result with training. This is ok.

```
from autokeras.utils import pickle_from_file
autokeras_model = pickle_from_file("best_autokeras_model.h5")
autokeras_score = autokeras_model.evaluate(x_test, y_test)
print(autokeras_score)

autokeras_predictions = autokeras_model.predict(x_text)
```

0.6623014767344664

I also check autokeras_predictions and y_test. Really 66.23% are same.

2- Loading keras model crashes.

```
from keras.models import load_model
keras_model = load_model('best_keras_model.h5')

keras_predictions = keras_model.predict(x_test)

keras_predictions_exact = []

for i in range(0, len(keras_predictions)):
    keras_predictions_exact.append(np.argmax(keras_predictions[i]))
```

It seems the exporting regular keras model doesn't export final weights of the model. Is this a known act? I should re-train it?

**Comment by crypdick** [comment]
Potentially related to https://github.com/keras-team/autokeras/issues/186#issuecomment-420676140
